SamplesDashboard/SamplesDashboardTests/RepositoriesServiceTests.cs

```diff
@@ -101,27 +153,13 @@ public async Task ShouldGetNullDependencies()
             var name = "powershell-intune-samples";
 
             //Act
-            var dependencies = await _sampleService.GetRepository(name);
+            var dependencies = await _repositoriesService.GetRepository(name);
```

Comment:
rename `dependencies` to `repository` as the function returns a repository object